src/azure-cli-core/setup.py

```diff
@@ -84,12 +86,7 @@
     url='https://github.com/Azure/azure-cli',
     zip_safe=False,
     classifiers=CLASSIFIERS,
-    packages=[
-        'azure.cli.core',
-        'azure.cli.core.commands',
-        'azure.cli.core.extension',
-        'azure.cli.core.profiles',
-    ],
+    packages=find_packages(exclude=["*.tests", "*.tests.*", "tests.*", "tests", "azure", "azure.cli"]),
```

Comment:
Change the package finding logic so that `vendored_sdks/` are included. Thank @arrownj for the help.